package.json

```diff
@@ -1,32 +1,74 @@
 {
-  "name": "ResearchShare-Contracts",
-  "version": "0.0.0",
-  "main": "index.js",
-  "author": "Ersikan <julien.philippon@epitech.eu>",
-  "license": "MIT",
+  "name": "reactrs",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/react": "^11.4.1",
+    "@material-ui/core": "^4.12.3",
+    "@material-ui/icons": "^4.11.2",
+    "@metamask/providers": "^8.1.1",
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "@types/jest": "^26.0.15",
+    "@types/node": "^12.0.0",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
+    "@types/tape": "^4.13.2",
+    "@typescript-eslint/eslint-plugin": "^4.29.2",
+    "@typescript-eslint/parser": "^4.29.2",
+    "babel-eslint": "^10.1.0",
+    "bootstrap": "^5.0.2",
+    "eslint-config-react-app": "^6.0.0",
+    "eslint-plugin-flowtype": "^5.9.0",
+    "eslint-plugin-import": "^2.24.0",
+    "eslint-plugin-jest": "^24.4.0",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-react": "^7.24.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-plugin-testing-library": "^4.11.0",
+    "ipfs-http-client": "^52.0.1",
+    "iso-url": "^1.1.5",
+    "it-all": "^1.0.5",
+    "it-concat": "^2.0.0",
+    "js-file-download": "^0.4.12",
+    "metamask-react": "^2.0.2",
+    "react": "^17.0.2",
+    "react-bootstrap": "^2.0.0-beta.4",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "react-spinners": "^0.11.0",
+    "typescript": "^4.1.2",
+    "web-vitals": "^1.0.1",
+    "web3": "^1.5.1",
+    "zokrates-js": "^1.0.34"
+  },
   "scripts": {
-    "ganache": "ganache-cli --deterministic",
-    "migrate": "truffle migrate",
-    "test": "truffle compile && ts-mocha --exit --recursive --extension js,ts",
-    "watch": "yarn test --watch",
-    "remix": "remixd -s $(pwd) --remix-ide https://remix.ethereum.org",
-    "zokrates": "node zokrates/zokrates.js"
+    "start": "yarn --cwd ./src/contracts build && react-app-rewired start",
+    "build": "yarn --cwd ./src/contracts build && react-app-rewired build",
+    "eject": "react-app-rewired eject",
+    "test": "mocha"
```

Comment:
Why is the test script only `mocha` now ?

Reply:
It now invoke test script on contract and does it too with ts-mocha for react env.